tool/tctl/common/token_command.go

```diff
@@ -257,11 +257,13 @@ func (c *TokensCommand) Add(ctx context.Context, client auth.ClientI) error {
 		if len(proxies) == 0 {
 			return trace.NotFound("cluster has no proxies")
 		}
+		setRoles := strings.ToLower(strings.Join(roles.StringSlice(), "\\,"))
```

Comment:
Can we simplify this and remove the extra quotes?
Roles are `[a-z]` so we should be able to just join them by `,`
Can we please add a test for this?

Reply:
Because of the way Helm parses values it should be in the form `--set roles="kube\,app\,discovery"`, it seems nothing else works. I've added a test checking this.